VulkanHppGenerator.cpp

```diff
@@ -13957,6 +14062,21 @@ void VulkanHppGenerator::readRequireEnum(
             m_constants[name] = { type, value, line };
           }
         }
+
+        // if we have a name and value, insert to enumConstants map
+        if ( attributes.contains( "name" ) && attributes.contains( "value" ) )
+        {
+          /** If I do readEnumsConstants(), it would require that I move "type" from the `required` map to the `optional` map at
```

Comment:
@asuessenbach, thoughts on this bit?

Reply:
Yep, I agree. Storing those name/value pairs in a separate bucket is appropriate, as they are very special.